README.md

```diff
@@ -124,7 +218,76 @@ Include it in your project like this:
     }
 
 
-## Hashes
+## Coding Guidelines
+
+Unfortunately we cannot use all Java 8 features, in order to be compatible with
+Android API <24. Please avoid using the following APIs:
```

Comment:
In Threema we're on minSdkVersion 16, btw :wink:

Reply:
Yup but IIRC we can still use API level 24?

Reply:
If a runtime feature is only supported in API level 24, and not in API level 16, then we cannot use it.

Many of the Java 8 features are not in the runtime though (e.g. lambdas).

Reply:
I copy pasted that from https://github.com/saltyrtc/saltyrtc-client-java, so I don't mind changing it but it needs to be done consistently.

Reply:
Yep. I think the comment stems from the fact that API level 24 added some Java 8 runtime features, so it's technically correct. Java 7 OTOH was fully supported for a long time (<16). I think we can keep it this way.